dgl_ptm/dgl_ptm/config.yaml

```diff
@@ -21,7 +21,7 @@ steering_parameters:
   characteristic_distance: 35
```

Comment:
We now have duplication in the parameters, as weight_a and weight_b are the deprecated names for `homophily_parameter` and `characteristic_distance`

Reply:
They are removed.